iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/LegConnectionType.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+/**
+ * @author Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public enum LegConnectionType {
+    Y,
```

Comment:
Just as information, CGMES uses `Yn` instead of `Y_GROUNDED`. Also note that in the future, we will have to add `Z` and `Z_GROUNDED` (or `Zn`).